docs/releases.md

```diff
@@ -0,0 +1,10 @@
+# CLR Instrumentation Engine Releases
+
+The below links provide the latest released packages for CLRIE. Some Azure platforms already support CLRIE as a first part extension. (see [here](scenarios/azure.md) for details).
+
+## Current Release - 1.0.31
+
+|Platform|Links|
```

Comment:
|Platform|Installer|Merge Module| ?

Then you can just change the links to [Download]

Reply:
I was trying to figure out a way to format it properly. I like your suggestion